resources/views/layouts/gallery.blade.php

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE HTML>
-<html lang="{{ $locale['language'] }}" style="visibility: hidden;">
+<html lang="{{ str_replace('_', '-', Lang::get_code()) }}" style="visibility: hidden;">
```

Comment:
Why this replacement? Previously, `$locale['lanuage']` has been equal to `$lang['language'] = Configs::getValueAsString('lang');` from `IndexController` and I don't see a similar replacement there. So why now?

Reply:
`Configs::getValueAsString('lang')` returns the same as `Lang::get_code()`.
This is just uniforming with `layouts/app.blade.php`.